app/src/main/res/layout/fragment_map.xml

```diff
@@ -73,12 +73,23 @@
                 android:src="@drawable/ic_social_settings" />
         </LinearLayout>
 
+        <LinearLayout
+            android:id="@+id/ll_battery"
+            android:layout_width="@dimen/margin_padding_size_36"
+            android:layout_height="@dimen/margin_padding_size_20"
+            android:layout_marginBottom="-10dp"
+            android:background="@drawable/blue_16dp_background"
+            android:orientation="vertical"
+            app:layout_constraintBottom_toTopOf="@+id/ll_traffic_light"
+            app:layout_constraintEnd_toEndOf="@+id/ll_traffic_light"
+            app:layout_constraintStart_toStartOf="@+id/ll_traffic_light"></LinearLayout>
```

Comment:
Reformat.
At line 85 you have the "</LinearLayout>" tag.
Maybe move the tag one line down?